src/handler.js

```diff
@@ -114,11 +116,45 @@ const handle = async message => {
       Authorization: `Bearer ${process.env.ACCESS_TOKEN}`,
     },
   }).then(response => {
-    response.data.pipe(resizer).pipe(writeStream);
+    response.data.pipe(resizer).pipe(resizedStream);
+  });
+
+  let b64Chunks = [];
+
+  resizedStream.pipe(b64Stream);
+
+  b64Stream.pipe(new Base64Encode());
```

Comment:
Having asked what you're doing with this lib at the top, it looks like you just pipe it through but then you're collecting the chunks anyway. are you sure you need the lib you included?